psscriptanalyzer/Test-Connection.ps1

```
@@ -0,0 +1,52 @@
+$Global:ConnectionResults = @()
+
+function Test-DatabaseConnection {
+    param(
+        [string]$Server,
+        [string]$Database,
+        [string]$Username,
+        [string]$Password
+    )
+    $connectionString = "Server=$Server;Database=$Database;User Id=$Username;Password=$Password;"
+    Write-Host "testing $connectionString"
+    $secure = ConvertTo-SecureString -String $Password -AsPlainText -Force
```

PSScriptAnalyzer (1.25.0)

[error] File 'Test-Connection.ps1' uses ConvertTo-SecureString with plaintext. This will expose secure information. Encrypted standard strings should be used instead.

(PSAvoidUsingConvertToSecureStringWithPlainText)